packages/core/src/fields/tests/test-fixtures.ts

```diff
@@ -93,7 +93,7 @@ export const filterTests = (withKeystone: any) => {
   );
 
   test(
-    'Filter: in - empty list',
```

Comment:
@mitchellhamilton I am really really unsure what this one should be - like what the hell?

Reply:
"list" here means a GraphQL list (which in js is an array) so it should be list.